src/main/java/org/tron/core/db/Manager.java

```diff
@@ -207,6 +218,13 @@
   @Getter
   private ForkController forkController = ForkController.instance();
 
+  private ExecutorService executorService = Executors
+      .newFixedThreadPool(1, r -> new Thread(r, "PreValidateTransactionSign"));
+
+  public static Map<ByteArrayWrapper, AccountCapsule> accountCache = new HashMap<>();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this "public static accountCache" field final ([squid:S1444](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1444))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1916&issues=AWgnKDa4rt4Kct_4Nf2E&open=AWgnKDa4rt4Kct_4Nf2E)

[ISSUE_KEY]: AWgnKDa4rt4Kct_4Nf2E